src/Symfony/Component/Cache/Traits/RedisClusterNodeProxy.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Traits;
+
+use \RedisCluster;
```

Comment:
please remove this and use `\RedisCluster` in the code

Reply:
Fixed 👍

Reply:
I think I added this originally trying to get psalm to pass.